### ISSUE TITLE
RSS feeds are missing from /channel/ profiles

### ISSUE THREAD

**Issue by Yowlen** [TeamNewPipe/NewPipe#585]
[X] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
[X] I checked if the issue/feature exists in the latest version.
User profiles & channel profiles look the same to the viewer, but apparently, they're two separate things within YouTube. I'm not sure what the difference is, but NewPipe doesn't process RSS feeds for channels like it does for users. An easy example is Outside Xbox vs Outside Xtra:
https://www.youtube.com/user/outsidexbox
https://www.youtube.com/channel/UCjf6YzmyaKi8880IXMJ5kGA

**Comment by mauriciocolli** [comment]
That issue will be fixed in the next release (when the pr is merged).

**Comment by Yowlen** [comment]
Umm, sorry. I never got around to testing this until now. This bug is still present. The /channel/ link above for Outside Xtra still doesn't show an option for an RSS feed.

**Comment by mauriciocolli** [comment]
This current version still don't use the new release of NewPipeExtractor.

**Comment by Yowlen** [comment]
Oh, ok. Thanks for the quick response. Shouldn't this be left open until it's in NewPipe? It seems confusing to close it without actually being in the final release.

**Comment by mauriciocolli** [comment]
Yes it is, the issue was "programmed" to be closed as soon as the pr was merged, but is better to be open until is actually fixed.
